avro/src/main/scala/hydra/avro/registry/SchemaRegistry.scala

```diff
@@ -185,6 +193,13 @@ object SchemaRegistry {
         }.toOption
       }
 
+      /**
+        * blah fill in
```

Comment:
Don't need the comment in the implementation, just in the trait itself

Reply:
Outdated, removed